src/app/external-cluster-wizard/steps/external-cluster/provider/aks/template.html

```diff
@@ -0,0 +1,171 @@
+<!--
+Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<form [formGroup]="form"
+      fxLayout="row"
+      fxLayoutGap="30px"
+      fxLayout.md="column"
+      fxLayoutGap.md="30px"
+      fxLayout.sm="column"
+      fxLayoutGap.sm="30px">
+  <div fxFlex="50%"
+       fxLayout="column"
+       fxLayoutGap="8px">
+    <mat-card-header class="km-no-padding">
+      <mat-card-title>Cluster</mat-card-title>
+    </mat-card-header>
+
+    <mat-form-field>
+      <mat-label>Name</mat-label>
+      <input matInput
+             type="text"
+             autocomplete="off"
+             [formControlName]="Controls.Name" />
+      <button mat-icon-button
+              type="button"
+              matSuffix
+              class="km-randomize-btn"
+              matTooltip="Generate name"
+              (click)="generateName()">
+        <i class="km-icon-randomize"></i>
+      </button>
+      <mat-hint>Enter a unique name for this cluster</mat-hint>
+      <mat-error *ngIf="form.get(Controls.Name).hasError(ErrorType.Required)">
+        Name is <strong>required</strong>.
+      </mat-error>
+    </mat-form-field>
+
+    <mat-form-field>
+      <mat-label>Kubernetes Version</mat-label>
+      <input [formControlName]="Controls.KubernetesVersion"
+             matInput
+             type="text"
+             autocomplete="off" />
+
+      <mat-hint>
+        Set the kubernetes version see list of
+        <a href="https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli"
+           rel="noopener noreferrer"
+           target="_blank">supported versions</a>.
+      </mat-hint>
+
+      <mat-error *ngIf="form.get(Controls.KubernetesVersion).hasError(ErrorType.Required)">
+        Version is <strong>required</strong>.
+      </mat-error>
+    </mat-form-field>
+
+    <mat-form-field>
+      <mat-label>Location</mat-label>
+      <input matInput
+             type="text"
+             autocomplete="off"
+             [formControlName]="Controls.Location" />
+      <mat-hint>Enter a region name for your cluster.</mat-hint>
+      <mat-error *ngIf="form.get(Controls.MinCount).hasError(ErrorType.Required)">
+        Location is <strong>required</strong>.
+      </mat-error>
+    </mat-form-field>
+
+    <mat-form-field>
+      <mat-label>Resource group</mat-label>
```

Comment:
```suggestion
      <mat-label>Resource Group</mat-label>
```

Update the error message too.